src/server/examples/examples_test.go

```diff
@@ -73,3 +74,156 @@ func TestExampleTensorFlow(t *testing.T) {
 	require.NoError(t, c.DeleteRepo("GoT_train", false))
 	require.NoError(t, c.DeleteRepo("GoT_scripts", false))
 }
+
+func TestFruitStand(t *testing.T) {
+
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+
+	c := getPachClient(t)
+	t.Parallel()
+
+	require.NoError(t, c.CreateRepo("data"))
+	repoInfos, err := c.ListRepo(nil)
+	require.NoError(t, err)
+	var repoNames []interface{}
+	for _, repoInfo := range repoInfos {
+		repoNames = append(repoNames, repoInfo.Repo.Name)
+	}
+	require.OneOfEquals(t, "data", repoNames)
+
+	cmd := exec.Command(
+		"pachctl",
+		"put-file",
+		"data",
+		"master",
+		"sales",
+		"-c",
+		"-f",
+		"https://raw.githubusercontent.com/pachyderm/pachyderm/master/doc/examples/fruit_stand/set1.txt",
```

Comment:
Yeah I think I'd still prefer using the local files. Fwiw we do have `TestPutFileURL` which tests the request for a `PutFile` with a URL. We just don't have a test that the CLI creates such a request properly. But I think we'll have that soon and decreasing the fragility seems worth it to me.